packages/gallery/src/components/gallery/blueprintsIndex.js

```diff
@@ -16,8 +16,22 @@ export default class BaseGallery extends React.Component {
     let galleryProps = { ...otherProps, styles: _styles, eventsListener: _eventsListener, domId};
 
 
-    if(this.props.useBlueprints) {
-      //
+    if (this.props.useBlueprints) {
+      try {
+        const {layoutFixer} = window;
```

Comment:
place this const under the typeof window !== 'undefined' if you are checking it otherwise it fails before the if (IMO the if can be split).